objects/src/accounts/storage/testing.rs

```diff
@@ -27,11 +27,13 @@ impl AccountStorageBuilder {
         self
     }
 
+    #[allow(dead_code)]
     pub fn add_map(&mut self, map: StorageMap) -> &mut Self {
         self.maps.push(map);
         self
     }
 
+    #[allow(dead_code)]
     pub fn add_maps<I: IntoIterator<Item = StorageMap>>(&mut self, maps: I) -> &mut Self {
         self.maps.extend(maps);
         self
```

Comment:
If these are not being used I'd delete them

Reply:
Yeah, I wasn't sure if there would be use for these after I was finished moving code, I might delete them later